.automatedtesting.json

```diff
@@ -12,10 +12,5 @@
     "ENABLE_GEMS": [
     ],
     "REPOS": [
-        {
-            "NAME": "o3de-extras",
-            "URL": "https://github.com/o3de/o3de-extras.git",
-            "BRANCH": "development"
-        }
```

Comment:
This change is OK as we are merging into `main` and not `development`.

As a matter of fact we don't want the o3de `main` branch to try to build any Gems in the o3de-extras `development` branch at all, so the change is actually beneficial to `main`.

PR https://github.com/o3de/o3de/pull/16857 is the only one where the `.automatedtesting.json` and `.gitnore` change shouldn't be merged to o3de `development`.

Reply:
Good catch, thank you! I'll remove that commit from the merge to development PR.